src/store/actions.ts

```diff
@@ -295,24 +294,12 @@ export const setAddressSuggestions = ({ commit }, addressSuggestions: any[]): vo
 }
 
 /**
- * Makes API calls to verify that the Corp Num is valid.
+ * Searches Entities or COLIN to fetch a business' info.
+ * @param corpNum the business identifier to search for
  * @returns a resolved promise on success or a rejected promise on failure
  */
 // FUTURE: not an action - move it to another module?
-export const fetchCorpNum = async ({ getters }, corpNum: string): Promise<BusinessFetchIF> => {
-  // NB: MRAS search was done on first page
-  // if (getters.getShowCorpNum === CorpNumRequests.MRAS) {
-  //   return checkMRAS({ getters }, corpNum)
-  // }
-
-  // *** FUTURE: COLIN search (business lookup) should have been done on first page
-  if (getters.getShowCorpNum === CorpNumRequests.COLIN) {
-    return checkCOLIN({ getters }, corpNum)
-  }
-}
-
-// FUTURE: not an action - move it to another module?
-async function checkCOLIN ({ getters }, corpNum: string): Promise<BusinessFetchIF> {
```

Comment:
Here I simply combined fetchCorpNum and checkCOLIN (and renamed it to searchBusiness).